### PR TITLE
chore: update PULL_REQUEST_TEMPLATE

### PR DESCRIPTION
**Pull request by YoshinoriN**
## Check List

**Please read and check followings before submitting a PR.**

- [x] Others (Update, fix, translation, etc...)

## Others

As a transitory measure for #1114

### CODE REVIEW

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -11,9 +11,11 @@
     - [ ] Screenshot filename is same as value of `name`.
     - [ ] Screenshot size is `800 * 500`.
     - [ ] Screenshot file format is `png`.
+    - [ ] My theme name is unique (not duplicate)
```
Comment (curbengh) [inline .github/PULL_REQUEST_TEMPLATE.md]: I think
```
`name` is unique
```
is sufficient

Comment (YoshinoriN) [reply .github/PULL_REQUEST_TEMPLATE.md]: Done

Comment (curbengh) [reply .github/PULL_REQUEST_TEMPLATE.md]: I meant just need three words, similar in style with
```
[ ] `link` URL is correct.
[ ] `preview` URL is correct.
```

Comment (YoshinoriN) [reply .github/PULL_REQUEST_TEMPLATE.md]: You means...

```diff
- - [ ] `link` and URL is correct.
- - [ ] `preview` URL is correct.
+ - [ ] `link` and `preview` URL is correct.
```

Comment (SukkaW) [reply .github/PULL_REQUEST_TEMPLATE.md]: @YoshinoriN @curbengh 

`link` is the project link of the theme, in most cases is GitHub Repo link, sometimes could be theme's own website (such as https://theme-next.org and https://theme-suka.skk.moe/en/).
`preview` is URL for the demo.

IMHO, we should merge 2 `preview` items instead:

```diff
- - [ ] `preview` URL is correct.
- - [ ] `preview` URL web site is rendered correctly.
+ - [ ] `preview` URL is correct and website is rendered correctly.
```

Comment (curbengh) [reply .github/PULL_REQUEST_TEMPLATE.md]: @YoshinoriN 
I mean
```
    - [ ] I have read the [theme publishing doc](https://hexo.io/docs/themes#Publishing).
    - [ ] `name` is unique.
    - [ ] `link` URL is correct.
    - [ ] `preview` URL is correct.
    - [ ] `preview` URL web site is rendered correctly.
    - [ ] Add a screenshot to `source/themes/screenshots`.
    - [ ] Screenshot filename is same as value of `name`.
    - [ ] Screenshot size is `800 * 500`.
    - [ ] Screenshot file format is `png`.
```

---

@SukkaW 
I agree with keeping existing items, I didn't mean removing any. I prefer not to merge to emphasize each point; `and` is not as strong as separate check.

Comment (YoshinoriN) [reply .github/PULL_REQUEST_TEMPLATE.md]: @SukkaW 
Thank you for your review and advice :)
But, I have the same opinion @curbengh

**Pull request merged**